src/components/KoroSection.scss

```diff
@@ -0,0 +1,30 @@
+@import '../../node_modules/open-city-design/src/scss/helsinki/mixins';
+
+.section-koro-top {
+  &.fog:after {
+    @include koro('basic', #9fc8eb, 400);
```

Comment:
We can get the Colors from _colors... If we stick with this method, something we need to change later on.